print/color.go

```diff
@@ -9,47 +9,64 @@ import (
 var (
 	// mu is used to synchronize writes from multiple goroutines.
 	mu sync.Mutex
+	// DisableOutput disables all output.
+	DisableOutput bool
+)
+
+func printf(fn func(string, ...interface{}), format string, a ...interface{}) {
```

Comment:
Please rename to `maybePrintf` and `maybePrintln`. Otherwise there'll be hell of a head-scratching when `printf` does not print :joy: